### DISCUSSION TITLE
Book and custom page layout: page title seems forced to stay in a fixed position. How to avoid it?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#8114] [Q&A]
### Description

Hi,
I have created and hello world custom layout:
- the rendered page https://aborruso.quarto.pub/sample/custom.html
- the code in the small repo https://github.com/aborruso/sample/blob/main/custom.qmd

The code is the one you see below, and I have inserted the title (`# Custom {.unnumbered}`) inside the div `.content-block`.

````
::: {.content-block}

# Custom {.unnumbered}

::: {.grid}


::: {.g-col-6}
This column takes 1/2 of the page
:::

::: {.g-col-6}
This column takes 1/2 of the page
:::

:::

:::
`````

But if I open the HTML code it is outside it. Is normal? Is there a way to avoid it?

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/c17b0878-00f2-46e8-b392-f520216462ad)

**Comment by mcanouil** [comment]
Since you did not provide a title, the first header was used as the title by Quarto.
Note that the layout is for the body part of the page.
This happens because it's a Book/Website and each page needs a title.

See a simple HTML document:

````qmd
---
format:
  html:
    page-layout: custom
    include-in-header:
      text: |
        <style type="text/css">
        .content-block {
          padding-top: 20px;
          padding-bottom: 10px;
          margin-left: 30px;
          margin-right: 30px;
        }
        </style>
---

::: {.content-block}

# Custom {.unnumbered}

::: {.grid}


::: {.g-col-6}
This column takes 1/2 of the page
:::

::: {.g-col-6}
This column takes 1/2 of the page
:::

:::

:::
````

**Reply by aborruso** [reply]
Thank you @mcanouil I have added a title, and now [I have](https://aborruso.quarto.pub/sample/custom.html) 1) the title and 2) the breadcrumbs.

My goal is, only for this single page, to have a special page, without 1) e 2).

Should I use CSS, and put both in hidden? Or are there parameters provided for custom layouts to not have title and breadcrumbs?

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/44386792-5883-4f02-a462-a32c93f2c82b)

**Reply by mcanouil** [reply]
Until #8108 is solved, the CSS workaround is the only way for the bread crumbs.

Regarding the title, either JS or CSS should work.
See the "title" definition for books: https://quarto.org/docs/books/book-structure.html#titles
